tests/test_create.py

```diff
@@ -2177,104 +2177,117 @@ def side_effect(self, url, **kwargs):
 
 
 @pytest.mark.skipif(on_win, reason="python doesn't have dependencies on windows")
-def test_disallowed_packages():
-    with make_temp_env() as prefix:
-        with env_var(
-            "CONDA_DISALLOWED_PACKAGES",
-            "sqlite&flask",
-            stack_callback=conda_tests_ctxt_mgmt_def_pol,
-        ):
-            with pytest.raises(CondaMultiError) as exc:
-                run_command(Commands.INSTALL, prefix, "python")
+def test_disallowed_packages(
+    tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture, monkeypatch: MonkeyPatch
+):
+    with tmp_env() as prefix:
+        monkeypatch.setenv("CONDA_DISALLOWED_PACKAGES", "sqlite&flask")
+        reset_context()
+        assert context.disallowed_packages == ("sqlite", "flask")
+        with pytest.raises(CondaMultiError) as exc:
+            conda_cli("install", f"--prefix={prefix}", "python", "--yes")
         exc_val = exc.value.errors[0]
         assert isinstance(exc_val, DisallowedPackageError)
         assert exc_val.dump_map()["package_ref"]["name"] == "sqlite"
```

Comment:
`conda-forge::python` doesn't depend on `sqlite` so let's use a different package that both `main::python` and `codna-forge::python` depend on:

```suggestion
        monkeypatch.setenv("CONDA_DISALLOWED_PACKAGES", "openssl&flask")
        reset_context()
        assert context.disallowed_packages == ("openssl", "flask")

        with pytest.raises(CondaMultiError) as exc:
            conda_cli("install", f"--prefix={prefix}", "python", "--yes")
        exc_val = exc.value.errors[0]
        assert isinstance(exc_val, DisallowedPackageError)
        assert exc_val.dump_map()["package_ref"]["name"] == "openssl"
```